jsonnet/openebs-mixin/dashboards/openebs/zfslocalpv.json

```diff
@@ -0,0 +1,1245 @@
+{
```

Comment:
Just recollecting.. why do we need to check-in the same file twice?

Reply:
To include the dashboard for helm installation, we are generating a dashboard using mixin.